contracts/interfaces/IERC6551Registry.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+interface IERC6551Registry {
```

Comment:
We will not have a custom registry deployed. This Interface is included in our codebase so that I can have proper solidity typing in `DecentHats` when i call the registry to create new Smart Account instances.